### DISCUSSION TITLE
Wait for loading or use basic interface

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#840] [General]
In the older versions, the prompt "Wait for loading or use basic interface" followed by the redirect to the actual file server wasnt there.  Is there anyway to force hfs to load the file server without seeing this message?

**Comment by rejetto** [comment]
there's no option for that at the moment.
That message is meant for older browsers, so if it can be nice to hide it, maybe it could be done in automatic way.
I will see to hide it if a modern browser is detected.
I would still keep a "Loading" message anyway, as it could take a bit on a very slow network.
